lib/active_record_shards/configuration_parser.rb

```diff
@@ -29,10 +29,13 @@ def explode(conf)
           conf["#{env_name}_replica"] = replica_conf
         end
 
-        if replica_conf = env_config.delete('slave')
+        # rubocop:disable Style/Next
```

Comment:
Rubocop is complaining bout not using `next` here, but I'm not sure if `next` makes sense in this case
```
lib/active_record_shards/configuration_parser.rb:32:9: C: Style/Next: Use next to skip iteration.
        if legacy_replica_conf = env_config.delete('slave')
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```